sourcemash/frontend/static/js/views/item_card_view.js

```diff
@@ -94,6 +95,15 @@ Sourcemash.Views.ItemCardView = Backbone.View.extend({
     },
 
     render: function() {
-        this.$el.html(this.template({ item: this.model }));
+        var shownCategories;
+        if (this.user.get('email')) {
+            shownCategories = this.model.categories.filter(function(category) {
+              return category.get("item_count") > 1;
+            });
+        } else {
+            shownCategories = this.model.categories.first(Sourcemash.Views.ItemCardView.MAX_CATEGORIES_SHOWN)
+        }
+
+        this.$el.html(this.template({ item: this.model, categories: shownCategories }));
     }
-});
+}, {MAX_CATEGORIES_SHOWN: 5});
```

Comment:
To confirm, this is being passed as a second dictionary to the entire view, which allows you to access it via the line ```Sourcemash.Views.ItemCardView.MAX_CATEGORIES_SHOWN```?